src/printer/index.js

```diff
@@ -608,12 +597,26 @@ function genericPrint(path, options, print) {
     case "arg_paren": {
       return path.call(print, "args");
     }
+
+    case "regexp_literal":
+      return concat([
+        "/",
+        join("", path.map(print, "content")),
+        "/",
+        path.call(print, "regex_end")
+      ]);
+
+    case "regexp_end":
+      return path.call(print, "regexp_end");
+
     case "string_literal": {
-      return concat(path.map(print, "string_content"));
+      const rawContent = n.string_content[0].content;
```

Comment:
This is wrong, but this will be fixed after:

- Add string interpolation support - https://github.com/iamsolankiamit/prettier-ruby/issues/8
- Support Ruby's handling of single quotes and double quote strings - https://github.com/iamsolankiamit/prettier-ruby/issues/20